adapters/openx/openx.go

```diff
@@ -140,17 +137,22 @@ func preprocess(imp *openrtb2.Imp, reqExt *openxReqExt) error {
 		imp.BidFloor = openxExt.CustomFloor
 	}
 
+	// copy imp.ext to outgoing request imp.ext, except for prebid fields
```

Comment:
commit message says "except prebid, bidder"

Reply:
I meant the prebid specific fields but I see how that didn't come out the way I intended